TestDateHelper failing due to DST issue

https://github.com/mickael-menu/zk/blob/1ec777de7ea4c8cafd5c42ed05ee94698529b925/internal/adapter/handlebars/handlebars_test.go#L244-L248

Hi! Package maintainer for Arch Linux here, ran into this when running tests.

Output:
```
=== RUN   TestDateHelper
    assert.go:44: Received (type string):
        "2009-11-17 20:34:58 +1300 NZDT"
    assert.go:45:
        ---
    assert.go:46: But expected (type string):
        "2009-11-17 20:34:58 +1200 NZST"
    assert.go:47:
        ---
    assert.go:48: Diff:
    assert.go:50:       "\"2009-11-17 20:34:58 +1300 NZDT\" != \"2009-11-17 20:34:58 +1200 NZST\""
--- FAIL: TestDateHelper (0.00s)
```

--- a/internal/adapter/handlebars/handlebars_test.go
+++ b/internal/adapter/handlebars/handlebars_test.go
@@ -243,8 +243,7 @@ func TestFormatDateHelper(t *testing.T) {
 
 func TestDateHelper(t *testing.T) {
 	context := map[string]interface{}{"now": time.Date(2009, 11, 17, 20, 34, 58, 651387237, time.UTC)}
-	localOffsetAndTZ := time.Now().Format("-0700 MST")
-	testString(t, "{{date \"2009-11-17T20:34:58\"}}", context, "2009-11-17 20:34:58 "+localOffsetAndTZ)
+	testString(t, "{{format-date (date \"2009-11-17T20:34:58\") 'timestamp'}}", context, "200911172034")
 }
 
 func TestShellHelper(t *testing.T) {

